### PR TITLE
Coin add

### PR DESCRIPTION
**Pull request by preetshah123**
Added in the feature to see your coins and other player coins. Also added some other simple stuff like if you dont have any cards, itll show "out of game." Also, added in another field to each player "inGame" which is set to 1 (wanted to use boolean but i didnt know how to do that cause i thought if i went update({inGame: "True"}) it would make it a string lol so idk but should be easy to fix cause I dont use this anywhere) when the game is started. I havent used it anywhere but i am thinking it might be easier to check this than if the card array length is 0 to see if someone is still in the game or not. Anyways ty babes

### CODE REVIEW

File: src/backend/game_logic.js [hunk 1]
```diff
@@ -29,11 +29,13 @@ export async function distributeCards(roomName) {
 		// give each player 2 cards and 2 coins
 		players.forEach(function(doc) {
 			let playerCards = [cards.pop(), cards.pop()];
-			playerCollection.doc(doc.id).update({cards: playerCards, coins: 2})
+			playerCollection.doc(doc.id).update({cards: playerCards, coins: 2, inGame: 1})
```
Comment (aravind-segu) [inline src/backend/game_logic.js]: `inGame: true` for boolean, just no quotes.

File: src/backend/startup.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ export async function createRoomName(roomName) {
 
 export async function startGame(roomName){
     await distributeCards(roomName).then(() => {
-        firestore.collection(root).doc(roomName).set({
+        firestore.collection(root).doc(roomName).update({
             startGame: true
```
Comment (aravind-segu) [inline src/backend/startup.js:40]: Similar to this for `inGame`

File: src/components/OtherPlayerInfoComponent.js [hunk 3]
```diff
@@ -16,14 +16,17 @@ function OtherPlayerInfo(props) {
...
-										{(props.data.cards.length >= 1) ? <PlayCard cardName={"back"}/> : <div/>}
+										{(props.data.cards.length >= 1) ? <PlayCard cardName={"back"}/> : <div>Out Of game, what a loser!</div>}
 									</Col>
 									<Col xs={6}>
 										{(props.data.cards.length == 2) ? <PlayCard cardName={"back"}/> : <div/>}
 									</Col>
 								</Row>
 							</Container>
 						</div>
+						<div align = "center" style = {{marginTop:10}}>
```
Comment (aravind-segu) [inline src/components/OtherPlayerInfoComponent.js]: use `em` and not just `10`. By default i think it uses `px` which can change by screen.

File: src/components/PlayCard.js [hunk 4]
```diff
@@ -33,6 +33,11 @@ function PlayCard(props) {
                 width: "8em",
             }}> <img src={ambassador} alt={props.cardName} style={{maxWidth: "100%", borderRadius: "1em"}} /></div >);
         }
+        case "undefined":{
```
Comment (aravind-segu) [inline src/components/PlayCard.js]: wait why do we need this

**Review by aravind-segu — CHANGES_REQUESTED** [review]

File: src/components/PlayCard.js (hunk 4 above)
Comment (preetshah123) [reply src/components/PlayCard.js]: oh forgot to delete nvm

**Review by aravind-segu — APPROVED** [review]

**Pull request merged**